integration/e2e_test.go

```diff
@@ -225,7 +225,7 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	require.NoError(s.T(), err)
 
 	// create Provider blockchain declaration
-	_, err = cli.TxCreateProviderExec(
+	res, err = cli.TxCreateProviderExec(
```

Comment:
noticed this, the `res` being used after was not related to this tx. I believe this was not expected. Can you confirm @troian ?

Reply:
good catch